### ISSUE TITLE
read.ctd.itp() is too hard-wired on data-column arrangement

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1029]
Below is the sort of code we have. It is clearly hard-wired to fit an individual file. Nobody has reported this as a problem, so I am marking this issue as low priority. I'll get to it sometime this summer.
```R
   if (isProfile) {
        ## %ITP 59, profile 2: year day longitude(E+) latitude(N+) ndepths
        ## 2013  247.25002   156.2163  80.3189  371
        ## %year day pressure(dbar) temperature(C) salinity oxygen(umol/kg)
        ## 2013  247.25036   18   -1.6548   30.5816  366.5573
        ## 2013  247.25043   20   -1.6523   30.7274  365.4786
        ## 2013  247.25052   22   -1.6537   31.1021  362.6732
        station <- gsub(":.*", "", gsub(".*profile[ ]*", "", lines[1]))
        d <- scan(text=lines[2], quiet=TRUE)
        year <- d[1]
        yearday <- d[2]
        longitude <- d[3]
        if (longitude < 0)
            longitude <- 360 + longitude
        latitude <- d[4]
        d <- read.table(text=lines[4:nlines])
        items <- scan(text=lines[3], what="character", quiet=TRUE)
        pcol <- grep("pressure", items)[1]
        Scol <- grep("salinity", items)[1]
        Tcol <- grep("temperature", items)[1]
        Ocol <- grep("oxygen", items)[1]
```

**Comment by dankelley** [comment]
A test data file is at `~/Dropbox/oce-working-notes/ctd/itp` but note that only the developers have access to this, since I've not checked on its sharing permission.

**Comment by dankelley** [comment]
Fixed in `develop` commit 9beade22c16bd4e5fa7b478c470433e82366efc0
